### DISCUSSION TITLE
has anyone got this working with svelte?

### DISCUSSION THREAD

**Discussion by diontron** [adonisjs/core#3579] [Help]
i'd like to use svelte instead of edge and have encore just take care of everything when i start the server. i do not want to use sveltekit seperately, i really prefer the folder structure of adonis and want to just be able to make `.svelte` files in my `resources/js/Pages` folder that i can point routes to with `inertia.render`

the idea is to use inertia as the bridge between adonis and svelte

i tried following this but it did not work properly: https://jyrki-grohn.medium.com/adonis-inertia-and-svelte-5291abedea65

i believe that is the closest i've gotten to making this work. has anyone got this working? i believe adonis + inertia + svelte might be the ultimate stack.

thank you

**Comment by mubashirjamali101** [comment]
Tried but no luck, I get error [ encore ] Error: output path no defined call: setOutputPath()...

**Comment by saumil-ventura1** [comment]
https://github.com/narad-muni/adonis-vite-svelte
See if this helps
